packages/checkout-extension/src/handlers/createReloadCheckoutHandler.ts

```diff
@@ -8,7 +8,12 @@ export function createReloadCheckoutHandler({
     return {
         commandType: ExtensionCommandType.ReloadCheckout,
         handler: () => {
-            void checkoutService.loadCheckout(checkoutService.getState().data.getCheckout()?.id);
+            void checkoutService.loadCheckout(checkoutService.getState().data.getCheckout()?.id, {
+                params: {
+                    // eslint-disable-next-line
+                    include: ['consignments.availableShippingOptions'] as any,
```

Comment:
```suggestion
                    include: [CheckoutIncludes.AvailableShippingOptions],
```

Reply:
I was going by this https://github.com/bigcommerce/checkout-js/blob/master/packages/core/src/app/checkout/Checkout.tsx#L203 but just tried to use the enum and seems to work fine. Seems like we have fixed the enum export issue overtime 😄

Reply:
For some reason this enum usage throws an undefined error where `CheckoutIncludes` is not defined. Might need to investigate it later.